db/models.py

```diff
@@ -1,4 +1,8 @@
+from django.core.exceptions import ValidationError
 from django.db import models
+from django.db.models import UniqueConstraint
+from django.contrib.auth.models import AbstractUser
+from settings import AUTH_USER_MODEL
```

Comment:
The import of `AUTH_USER_MODEL` should be from `django.conf` instead of `settings`. Use `from django.conf import settings` and then access `settings.AUTH_USER_MODEL`.